packages/e2e/cypress/integration/ui/components/authenticator/sign-up-with-email/sign-up-with-email.steps.ts

```diff
@@ -23,11 +23,11 @@ When('I type the email {string}', (email: string) => {
 });
 
 And('I type the password {string}', (password: string) => {
-  cy.findByLabelText(/^password$/i).type(Cypress.env(password));
+  cy.findByPlaceholderText(/^password$/i).type(Cypress.env(password));
```

Comment:
Cypress/testing-lib couldn't find the `label` field even though it's visually hidden?

https://github.com/aws-amplify/amplify-ui/blob/0e4fea4fdc3257193b20e4dcc1a1e6a080de2d97/packages/react/src/primitives/Label/Label.tsx#L15-L25

I guess that's by design?

Reply:
Yeah, seems that way. I had to make the same changes to the SignIn screen. I guess Cypress is only looking for what it is visually on the page?

Reply:
I found that testing-lib can't find it even when it's visible as well (I got the same error in the unit tests). There appears to be some bug in the library or possibly JSDOM, but I couldn't track it down

Reply:
@reesscot Did that testing include the fix to `arria-labelledby`?

From Slack today:

I think the problem is aria-labelledby using the input’s `id` instead of a label `id`:

```html
<input class="amplify-input amplify-field-group__control" id="amplify-field-0108a3625c5f" aria-invalid="false" required="" type="text" aria-labelledby="amplify-field-0108a3625c5f" name="username" placeholder="Fake placeholder" data-amplify-usernamealias="true" value="">
```

> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute